Patches/Kit's VFE Weapons/Ammo/Ammo_KitPlasma.xml

```diff
@@ -0,0 +1,150 @@
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Kit's VFE Weapons</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<!-- ==================== Boltcaster ========================== -->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+					<!-- ==================== AmmoSets ========================== -->
+
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_KIT_PlasmaPistol</defName>
+						<label>Plasma Pistol Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellPistol>Bullet_KIT_PlasmaPistol</Ammo_PlasmaCellPistol>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+					
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_KIT_PlasmaSMG</defName>
+						<label>Plasma Pistol Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellPistol>Bullet_KIT_PlasmaSMG</Ammo_PlasmaCellPistol>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+					
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_KIT_PlasmaRifle</defName>
+						<label>Plasma Rifle Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellRifle>Bullet_KIT_PlasmaRifle</Ammo_PlasmaCellRifle>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+					
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_KIT_PlasmaRifle</defName>
+						<label>Plasma Rifle Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellRifle>Bullet_KIT_PlasmaSniper</Ammo_PlasmaCellRifle>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+
+					<!-- ==================== Ammo ========================== -->
+
+					<!-- ================== Projectiles ================== -->
+
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseVFEIPlasmaBullet">
+						  <defName>Bullet_KIT_PlasmaPistol</defName>
+						  <label>plasma pistol shot</label>
+						  <graphicData>
+							<texPath>Things/Item/Weapons/Plasma_Beam</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+						  </graphicData>
+						  <projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageAmountBase>15</damageAmountBase>
+							<armorPenetrationSharp>10</armorPenetrationSharp>
+							<armorPenetrationBlunt>10</armorPenetrationBlunt>
+							<empShieldBreakChance>0.2</empShieldBreakChance>
+							<secondaryDamage>
+							  <li>
+								<def>EMP</def>
+								<amount>3</amount>
+							  </li>
+							</secondaryDamage>
+						  </projectile>
+						</ThingDef>
+						
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseVFEIPlasmaBullet">
+						  <defName>Bullet_KIT_PlasmaRifle</defName>
+						  <label>plasma rifle shot</label>
+						  <graphicData>
+							<texPath>Things/Item/Weapons/Plasma_Nail</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+						  </graphicData>
+						  <projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageAmountBase>30</damageAmountBase>

```

Comment:
I'm not sure how the plasma weapons from other patches in general compare, but this feels pretty punishing to be coming flying out of an automatic infantry rifle--it has a tendency to just completely blast away limbs with a single burst.

Reply:
I've dropped the automatic functionality from the rifle, since even with shotgun type burst, it was too much yeah. The original is single shot. While it's fairly high damage still, it is, compared to the VFE plasma options in the same catergory, stronger than both, but one plasma has 3/6 round bursts of 15 or so damage, and the other deals burn + stun damage.